e2e/tests/visual-output/rendering/index.ts

```diff
@@ -11,6 +11,11 @@ beforeRenderTest(
     return renderTests.map((renderTest) => {
       return {
         async boundFunction() {
+          if (renderTest.hasOwnProperty('enableGLFiltering')) {
+            fabric.config.configure({
+              enableGLFiltering: renderTest.enableGLFiltering,
```

Comment:
this could not work well if switched between tests.

Reply:
but this is what qunit did
have a look at this
 https://github.com/fabricjs/fabric.js/pull/10613/files#diff-0ea34a0a8eb6a9379c2e2241313e09483701f7c94a24ebf8060963424e9a2a05L3

Reply:
ok if each page import its own fabric library that saves it. i think the filter code doesn't get swapped if you disable webgl fitlering after a filtering operation